p2pserver/protocols/utils/govnode_resolver.go

```diff
@@ -162,18 +202,22 @@ func GetPeersConfig(backend *ledger.Ledger, view uint32) ([]*config.VBFTPeerStak
 	}
 
 	govCount := uint32(0)
-	var peerstakes []*config.VBFTPeerStakeInfo
+	var peerstakes []*GovNodeInfo
 	for _, id := range peerMap.PeerPoolMap {
+		isConsensus := id.Status == governance.ConsensusStatus || id.Status == governance.QuitConsensusStatus
 		switch id.Status {
 		case governance.CandidateStatus, governance.ConsensusStatus, governance.QuitConsensusStatus:
 			conf := &config.VBFTPeerStakeInfo{
 				Index:      uint32(id.Index),
 				PeerPubkey: id.PeerPubkey,
 				InitPos:    id.InitPos + id.TotalPos,
 			}
-			peerstakes = append(peerstakes, conf)
+			peerstakes = append(peerstakes, &GovNodeInfo{
+				ConsNode:          isConsensus,
```

Comment:
if `ConsNode` is used for voting check, `governance.QuitConsensusStatus` should not be ConsNode?

Reply:
`QuitConsensus ` means it will quit in next view, and is still consensus node in current view.